app/code/controller/CustomTaxonomyDirectoryController.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Controller;
+
+use Page;
+use SilverStripe\View\ArrayData;
+use SilverStripe\ORM\DataExtension;
+use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\GraphQL\Controller;
+use SilverStripe\Control\HTTPRequest;
+use SilverStripe\Taxonomy\TaxonomyTerm;
+use SilverStripe\Taxonomy\Controllers\TaxonomyDirectoryController;
+
+class CustomTaxonomyDirectoryController extends TaxonomyDirectoryController
+{
+    private static $allowed_actions = [
+        'showTags'
+    ];
+
+    /**
+     * This action allows for pages to be filtered by a desired term
+     *
+     * @param HTTPRequest $request
+     * @return void
+     */
+    public function showTags(HTTPRequest $request)
+    {
+        $termString = $request->param('ID');
```

Comment:
I think we can rename this variable to `$termID` so it matches the expected parameter type :)